karols-frontend/src/resume/getReservationData.test.js

```diff
@@ -1,8 +1,8 @@
-import { showPhone } from "."
+import getReservationData from "."
 
-describe("showPhone", () => {
-  it("should return false if no shop selected", () => {
-    const noSelectedShopState = {
+describe("getReservationData", () => {
+  it("should return an empty array if no data selected", () => {
```

Comment:
should return an empty object {} because expected result will always be an object